Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'govuk_taxonomy_helpers'
+
+node {
+  def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
+
+  try {
+    stage('Checkout') {
+      checkout scm
+    }
+
+    stage('Clean') {
+      govuk.cleanupGit()
+      govuk.mergeMasterBranch()
+    }
+
+    stage('Bundle') {
+      govuk.bundleApp()
```

Comment:
We found on another gem that we couldn't use `bundleApp`. I think the problem was that it adds the `--deployment` flag, which isn't appropriate for a gem.

We used `sh("bundle install --path ${JENKINS_HOME}/bundles/${JOB_NAME}")` instead but we should probably extract it to the Jenkins helpers in govuk-puppet.